kolibri/auth/test/test_users.py

```diff
@@ -60,12 +59,18 @@ class UserSanityTestCase(TestCase):
     Sanity checks basic functionality of user models.
     """
     def setUp(self):
-        self.user = FacilityUser.objects.create(username="mike", first_name="Mike", last_name="Gallaspy")
-        self.do = DeviceOwner.objects.create(username="bar")
-
-    def test_base_user(self):
-        with self.assertRaises(NotImplementedError):
-            BaseUser().is_device_owner()
+        self.dataset = FacilityDataset.objects.create()
+        self.user = FacilityUser.objects.create(
+            username="mike",
+            first_name="Mike",
+            last_name="Gallaspy",
+            password="###",
```

Comment:
Use [set_password](https://docs.djangoproject.com/en/1.9/ref/contrib/auth/#django.contrib.auth.models.User.set_password) unless you don't care about authentication. The password field is, IIRC, actually the hash of the raw password, so if you set it like this you'll won't be able to authenticate.